### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Script1988**
Cars_filter_and_search
driver_additional_info
admin_additional_info
admin_lisence_number

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.idea
+db.sqlite3
+.pyc
```
Comment (Abnormaltype) [inline .gitignore:3]: please, add a blank for the end of the file

File: taxi/admin.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Car, Driver
+
+# Register your models here.
```
Comment (Abnormaltype) [inline taxi/admin.py:6]: remove unnecessary code

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
image
remove pycache from the PR, also use `black` for your project

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
image
you still have the pycache files

**Comment by Script1988** [comment]
I don`t see __pycashe__ in my project... Should I change some settings to see it?
Pycashe

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
you should delete them from the PR)

**Pull request closed without merging**